azure_arc_servers_jumpstart/oci/ubuntu/terraform/main.tf

```diff
@@ -0,0 +1,162 @@
+variable "tenancy_ocid" {}
+variable "user_ocid" {}
+variable "fingerprint" {}
+variable "private_key_path" {}
+variable "compartment_ocid" {}
+variable "region" {}
+variable "ssh_public_key" {}
+
+resource "azurerm_resource_group" "azure_rg" {
+  name     = var.azure_resource_group
+  location = var.azure_location
+  tags = {
+    project = "jumpstart_azure_arc_servers"
+  }
+}
+
+variable "ad_region_mapping" {
+  type = map(string)
+  default = {
+    us-phoenix-1 = 2
+    us-ashburn-1 = 1
+  }
+}
```

Comment:
Consider moving variable declarations to standalone variables.tf file.

Reply:
@JerryMSFT this is a Terraform best practice. Please implement.